osxcollector/output_filters/virustotal/api.py

```diff
@@ -58,6 +58,22 @@ def get_file_reports(self, resources):
         responses = self._make_requests(self.BASE_DOMAIN + 'file/report', params)
         return dict([(resource, response) for resource, response in zip(resources, responses)])
 
+    def get_url_reports(self, resources):
```

Comment:
This wasn't very DRY before your change but now it's less DRY.
It would be nice to abstract the guts of this and it's neighboring methods into a single implementation they can all call into.